<ui version="4.0" >
 <class>activityDialogClass</class>
 <widget class="QDialog" name="activityDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Choose your activity</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../../../../../../:/icons/crystal_project/statuses.png</iconset>
  </property>
  <widget class="QLabel" name="captionEdit" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>20</y>
     <width>181</width>
     <height>60</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="verticalScrollBarPolicy" >
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy" >
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="generalList" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>80</y>
     <width>181</width>
     <height>94</height>
    </rect>
   </property>
   <property name="showDropIndicator" stdset="0" >
    <bool>false</bool>
   </property>
   <property name="movement" >
    <enum>QListView::Static</enum>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <property name="viewMode" >
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="specificList" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>170</y>
     <width>181</width>
     <height>94</height>
    </rect>
   </property>
   <property name="showDropIndicator" stdset="0" >
    <bool>false</bool>
   </property>
   <property name="movement" >
    <enum>QListView::Static</enum>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <property name="viewMode" >
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseButton" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>270</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Choose</string>
   </property>
   <property name="icon" >
    <iconset>../../../../../../../:/icons/crystal_project/apply.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>270</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
   <property name="icon" >
    <iconset>../../../../../../../:/icons/crystal_project/cancel.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../../../qutim.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>activityDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>170</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>194</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
